$OpenBSD: patch-contrib_vanilla-module_mk,v 1.1 2011/07/08 11:42:09 jasper Exp $

Add an 'all' target to allow building without running tests.

--- contrib/vanilla-module.mk.orig	Mon May  9 04:49:39 2011
+++ contrib/vanilla-module.mk	Fri Jul  8 13:39:44 2011
@@ -1,3 +1,4 @@
+all: $(MODULE).fasl
 
 $(MODULE).fasl: $(MODULE).lisp ../../output/sbcl.core
 	$(SBCL) --eval '(compile-file (format nil "SYS:CONTRIB;~:@(~A~);~:@(~A~).LISP" "$(MODULE)" "$(MODULE)"))' </dev/null
